README.md

```diff
@@ -85,10 +86,12 @@ failed to start docker container: Error response from daemon: Ports are not avai
   
 * `npm run build` to make an optimised build - you'll need to do this before running tests.
 
-* `npm run test` to run all tests. This will run both component and integration tests.
+* `npm run test` to run all tests. This will run Cypress component and integration and Jest component tests.
   * single out test suites with:
-    * `npm run test:component` for just component (unit) tests
-    * `npm run test:e2e` for just end-to-end tests
+    * `npm run test:component` for Cypress and Jest component (unit) tests
+    * `npm run test:e2e` for just Cypress end-to-end tests
```

Comment:
should this not be 'npm run cypress:e2e'? may have misunderstood but looks like you changed the command name in checks.yml. Could be wrong on this though lmk.

Reply:
test:e2e and cypress:e2e are technically equivalent where the npm scripts are defined, but it's probably more clear if I put 'npm run cypress:e2e'